docs/v1/P4Runtime-Spec.mdk

```diff
@@ -679,6 +679,22 @@ described as follows:
       controller is determined to be a slave for a given (`device_id`,
       `role_id`).
 
+The gRPC libraries enable a server process to cause a function to be called when
```

Comment:
This assumes that the gRPC P4Runtime service is used in synchronous mode. Maybe this should be rephrase in a more generic fashion (i.e. "gRPC lets the server identify which client originated each stream packet etc.") or we could indicate that we are talking about synchronous mode while stating that the same principle applies to asynchronous as well.

Reply:
I have attempted to address this comment, by starting with a sentence based on your suggestion, and keeping the rest of the paragraph as an example of how, in synchronous mode, gRPC enables this capability.